test/m1_facilitator/SeedDataInitializer.test.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 OpenST Ltd.
```

Comment:
Rename the file to `isValidSeedData.test.ts`.
This file would be under `test/m1_facilitator/SeedDataInitializer/` folder.